infrastructure/src/main/java/org/corfudb/infrastructure/log/InMemoryStreamLog.java

```diff
@@ -30,15 +30,13 @@
 public class InMemoryStreamLog implements StreamLog {
 
     private Map<Long, LogData> logCache;
-    private volatile long startingAddress;
```

Comment:
still in the case of sparse trim the starting address can change, isn't this relevant to keep track of?

Reply:
I don't know why Wenbin's change is in this PR. @WenbinZhu Could you help answer this question?

Reply:
This is not my change.

Reply:
I see why I delete this line. startingAddress is used for prefixTrim for streamLog. For the new GC, log data (as well as garbage data) are trimmed at the logunit's own pace. As as result, startingAddress is not needed for externally triggering compaction.